Dockerfile

```diff
@@ -7,6 +7,9 @@ COPY logdna.gpg /etc/
 RUN echo "deb http://repo.logdna.com stable main" > /etc/apt/sources.list.d/logdna.list && \
     apt-key add /etc/logdna.gpg && \
     apt-get -y update && \
-    apt-get -y install logdna-agent
-
+    apt-get -y install logdna-agent && \
+    apt-get -y upgrade apt && \
```

Comment:
It would probably be good to replace lines 11 and 12 with `apt-get -y upgrade && \`

Reply:
That may severely bloat the image and take a very long time. But we can give it a shot, why not just the 2 packages instead?

Reply:
My thought was to see what the impact was to the image, turns out it didnt really change the size at this time.  We are adding this tooling so that we can make releases happen more often to keep the image secure. If we have to adjust the package list on the regular because of new security vulnerabilities, we are just making toil for ourselves.